packages/debug/src/browser/console/debug-console-items.tsx

```diff
@@ -137,7 +137,7 @@ export class DebugVariable extends ExpressionContainer {
     constructor(
         session: DebugSessionProvider,
         protected readonly variable: DebugProtocol.Variable,
-        protected readonly parent: ExpressionContainer
+        public readonly parent: ExpressionContainer
```

Comment:
In general, we try to avoid writing `public`, as it's the default.

Reply:
I've changed this.